components/tyk-gateway/values.yaml

```diff
@@ -156,6 +156,9 @@ gateway:
   # The port which will be exposed on the container for tyk-gateway
   containerPort: 8080
 
+  scratch:
+    enabled: true
```

Comment:
from naming conventions, i would suggest a naming which will be `false` by default. For instance, renaming `scratch` as `detachInitContainer` and setting `detachInitContainer.enabled=false`. What do you think? 

In anyways, we need a descriptive comment in this section to describe the purpose of this field.

Reply:
The change is essentially about storing apps/policies/middleware on a mounted volume or inside the container FS. 
But I can't think up any concise name for this, so let it be `detachInitContainer`